chamber/models/__init__.py

```diff
@@ -217,6 +225,11 @@ def _get_save_extra_kwargs(self):
     def _pre_save(self, *args, **kwargs):
         pass
 
+    def _call_dispatcher_group(self, group_name, changed_fields, *args, **kwargs):
+        if hasattr(self, group_name):
+            for dispatcher in getattr(self, group_name):
+                dispatcher(self, changed_fields, *args, **kwargs)
```

Comment:
takže se tam posílá inst, changed_fields, a další argumenty to bych jen v dokumentaci zmínil případně udělal příklad dispatcheru